docs/package-inclusion-guide.md

```diff
@@ -0,0 +1,55 @@
+# Azure Cloud Shell Package Inclusion Guide
+
+## Background
+
+There are millions (or potentially billions) of Linux packages. This guide goes through the principles of what the Azure Cloud Shell team uses to consider what gets included in the service.
+
+
+
+## General Principles
+
+1. Useful for Azure users and administrators 
```

Comment:
Is there a way for each item to link to its expanded definition below?

Reply:
I guess we could. The links could get a bit tricky though. For example, for point 2, it links to the Azure Linux repo. If we also wanted to link to the second point (in the discussion of principles), it may get a bit confusing.